cogito/protocol.go

```diff
@@ -239,14 +247,33 @@ func (src *Source) Validate() error {
 	sinks := sets.From(src.Sinks...)
 	if sinks.Size() == 0 || sinks.Contains("github") {
 		// Cogito commit Github status mandatory fields.
+		// cannot specify both github_app and access_token
```

Comment:
Comments at lines 250 and 255 seem to be repeating the error messages just two lines below them.

Reply:
Fixed.